sycl/doc/extensions/experimental/sycl_ext_oneapi_native_math.asciidoc

```diff
@@ -0,0 +1,93 @@
+# Native math functions extension for DPC++: SYCL_ONEAPI_NATIVE_MATH
```

Comment:
We recently added a [template ](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/template.asciidoc)to use for extension specifications.  Please update this specification to use that template.  Also see the description in [README-process](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/README-process.md).

Reply:
Thank you for your prompt review @gmlueck!

I changed the template and addressed your comments in the next commit.